process/economics/export_test.go

```diff
@@ -0,0 +1,32 @@
+package economics
+
+import "math/big"
+
+// SetRewardsParameter -
+type SetRewardsParameter struct {
+	RewardsSettingEpoch              uint32
+	LeaderPercentage                 float64
+	ProtocolSustainabilityPercentage float64
+	ProtocolSustainabilityAddress    string
+	DeveloperPercentage              float64
+	TopUpGradientPoint               *big.Int
+	TopUpFactor                      float64
+}
+
+// GetRewardsSetting -
+func (ed *economicsData) GetRewardsSetting() *SetRewardsParameter {
```

Comment:
if this function would have returned a `config.EpochRewardSettings` value then all require.Equal from the previous test could have been written as:
```GO
require.Equal(t, rs[0], economicsData.GetRewardsSetting())
```

This kind of test should be favored because it tests all values automatically (including the new ones that might be added)